opendevin/runtime/docker/image_agnostic_util.py

```diff
@@ -60,7 +60,11 @@ def _build_sandbox_image(
 
 
 def _get_new_image_name(base_image: str) -> str:
-    return base_image + '_od_sandbox'
+    if ":" not in base_image:
```

Comment:
I did some polish. Please take a look if you have time. Thanks! Then we can merge this pr.
@yufansong